custom_components/tuya_local/devices/desk_lamp.yaml

```diff
@@ -0,0 +1,32 @@
+name: Desk Lamp
+products:
+  - id: b0j0lto1o0ijwb39
+    name: Desk Lamp Setti+ SL601
+primary_entity:
+  entity: light
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 2
+      type: string
+      name: color_mode
```

Comment:
Since this only has one valid value, I suggest renaming it (eg to "work_mode", matching Tuya terminology).  If it is called "color_mode", this integration will try to set it in some circumstances, but it is probably better that we leave it alone.  The presence of a color_temp dp, and absence of rgbhsv will make this light default to color_temp for supported_color_modes without any color_mode dp.

Reply:
More so now that I've read your log message which has this as "Breathe" - I suspect it is not color_mode at all, but effect.  But to implement it as "effect", we'll need all the valid values.